online_test/settings.py

```diff
@@ -122,20 +122,25 @@
 
 STATIC_ROOT = 'yaksh/static/'
 
+
+
+
 # Set this varable to <True> if smtp-server is not allowing to send email.
-EMAIL_USE_TLS = False
+EMAIL_USE_TLS = True
+
+EMAIL_HOST = 'smtp.gmail.com'
 
-EMAIL_HOST = 'your_email_host'
+EMAIL_PORT = '587'
 
-EMAIL_PORT = 'your_email_port'
+EMAIL_HOST_USER = 'yakshcertificate@gmail.com'
 
-EMAIL_HOST_USER = 'email_host_user'
+EMAIL_HOST_PASSWORD = 'elite2019'
 
-EMAIL_HOST_PASSWORD = 'email_host_password'
+EMAIL_USE_SSL = False
 
 # Set EMAIL_BACKEND to 'django.core.mail.backends.smtp.EmailBackend'
 # in production
-EMAIL_BACKEND = 'django.core.mail.backends.dummy.EmailBackend'
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
```

Comment:
Keeping this to smtp will require a email backend even in dev env. So keep it as dummy. On production server this will be changed.